### PR TITLE
Added struct for SEXS AVR

### PR DESCRIPTION
**Pull request by m-bossart**

### CODE REVIEW

File: src/descriptors/power_system_structs.json [hunk 1]
```diff
@@ -3545,6 +3545,92 @@
...
+        {
+          "name": "ext",
+          "data_type": "Dict{String, Any}",
+          "null_value": "Dict{String, Any}()",
+          "default": "Dict{String, Any}()"
+        },
+        {
+          "name": "states",
+          "exclude_setter": true,
+          "comment": "The states are:\n\tVf: Voltage field,\n\tVr: Lead-lag state",
```
Comment (jd-lara) [inline src/descriptors/power_system_structs.json:3607]: can you remove the break lines `\n`

**Review by jd-lara — CHANGES_REQUESTED** [review]
Please also add the export of SEXS in the file `PowerSystems.jl`

**Comment by jd-lara** [comment]
@m-bossart more testing is missing to merge the PR: 

see  https://nrel-siip.github.io/PowerSystems.jl/stable/code_base_developer_guide/adding_new_types/#Testing-the-addition-of-new-struct-to-the-code-base

**Comment by m-bossart** [comment]
Can you elaborate on this? Is this a test that I need to implement or that happens automatically? From what I can tell there is no method defined to add the new component to the system, but I'm not sure exactly what I need to do.

**Pull request merged**